src/main/java/ru/jma/userservice/model/BaseEntity.java

```diff
@@ -0,0 +1,15 @@
+package ru.jma.userservice.model;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
+public class BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long objectId;
```

Comment:
You and I agreed to use UUID instead of Long

Reply:
Michael showed you an example of UUID implementation on Skype

Reply:
@Id
@GeneratedValue(generator = "UUID")
@GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
@Column(name = "id", updatable = false, nullable = false)
private UUID id;

Cons of this approach:
- storing UUID values (16-bytes) takes more storage than integers (4-bytes) or even big integers(8-bytes)
- debugging seems to be  more difficult
- using UUID values may cause performance issues due to their size and not being ordered

Is it really necessary?